docs/tutorial/security.md

```diff
@@ -44,41 +44,495 @@ effort are always very welcome.
-`webview` tag and make sure to disable the `nodeIntegration`.
+Node.js integration enabled. Instead, use only local files (packaged together
+with your application) to execute Node.js code. To display remote content, use
+the [`webview`](web-view) tag and make sure to disable the `nodeIntegration`.
 
-#### Checklist
+#### Checklist: Security Recommendations
 
 This is not bulletproof, but at the least, you should attempt the following:
 
-* Only display secure (https) content
-* Disable the Node integration in all renderers that display remote content
-  (setting `nodeIntegration` to `false` in `webPreferences`)
-* Enable context isolation in all renderers that display remote content
-  (setting `contextIsolation` to `true` in `webPreferences`)
-* Use `ses.setPermissionRequestHandler()` in all sessions that load remote content
-* Do not disable `webSecurity`. Disabling it will disable the same-origin policy.
-* Define a [`Content-Security-Policy`](http://www.html5rocks.com/en/tutorials/security/content-security-policy/)
+* [Only display secure (https) content](#only-display-secure-content)
+* [Disable the Node integration in all renderers that display remote content](#disable-node-integration-for-remote-content)
+* [Enable context isolation in all renderers that display remote content](#enable-context-isolation-for-remote-content)
+* [Use `ses.setPermissionRequestHandler()` in all sessions that load remote content](#handle-session-permission-requests-from-remote-content)
+* [Do not disable `webSecurity`](#do-not-disable-websecurity)
+* [Define a `Content-Security-Policy`](#define-a-content-security-policy)
 , and use restrictive rules (i.e. `script-src 'self'`)
-* [Override and disable `eval`](https://github.com/nylas/N1/blob/0abc5d5defcdb057120d726b271933425b75b415/static/index.js#L6-L8)
+* [Override and disable `eval`](#override-and-disable)
 , which allows strings to be executed as code.
-* Do not set `allowRunningInsecureContent` to true.
-* Do not enable `experimentalFeatures` or `experimentalCanvasFeatures` unless
-  you know what you're doing.
-* Do not use `blinkFeatures` unless you know what you're doing.
-* WebViews: Do not add the `nodeintegration` attribute.
-* WebViews: Do not use `disablewebsecurity`
-* WebViews: Do not use `allowpopups`
-* WebViews: Do not use `insertCSS` or `executeJavaScript` with remote CSS/JS.
-* WebViews: Verify the options and params of all `<webview>` tags before they
-  get attached using the `will-attach-webview` event:
+* [Do not set `allowRunningInsecureContent` to `true`](#do-not-set-allowRunningInsecureContent-to-true)
+* [Do not enable experimental features](#do-not-enable-experimental-features)
+* [Do not use `blinkFeatures`](#do-not-use-blinkfeatures)
+* [WebViews: Do not use `allowpopups`](#do-not-use-allowpopups)
+* [WebViews: Verify the options and params of all `<webview>` tags](#verify-webview-options-before-creation)
+
+
+## Only Display Secure Content
```

Comment:
`Display` should be generalized to `Load` or something since there's plenty of resources (js files for instance) that don't necessarily get displayed